Jormungand.DiscordReader-Mouth/src/main/java/entities/JorgPerms.java

```diff
@@ -0,0 +1,13 @@
+package entities;
+
+import net.dv8tion.jda.api.Permission;
+
+import java.util.EnumSet;
+
+public class JorgPerms {
+    private long permissions;
+
+    public JorgPerms(long permission){
+        this.permissions = permission;
+    }
+}
```

Comment:
first look on how to handle it:
have it include the PermName and the boolean, e.g in a key-value store ala `MESSAGE_READ:TRUE`

Reply:
Second look: needs additional info if it's guild/channel and if it's voice/text